src_c/color.c

```diff
@@ -1753,6 +1758,16 @@ _color_float(pgColorObject *color)
     return PyFloat_FromDouble((double)tmp);
 }
 
+/**
+ * bytes(color)
+ */
+static PyObject *
+_color_bytes(pgColorObject *color)
+{
+    return PyBytes_FromFormat("%c%c%c%c", color->data[0], color->data[1],
+                              color->data[2], color->data[3]);
```

Comment:
```suggestion
    return PyBytes_FromStringAndSize((char *)color->data, 4);
```

I wonder if something like this can work. Would result in cleaner and a little bit faster code too I would guess.

Reply:
Cleaner? Don't know about that, it does look slicker though and I like it (and it seems to work)
A little bit faster? Maybe, though then again this is C... so does it matter?